### PR TITLE
fix create gateway

### PR DESCRIPTION
**Pull request by tomheaton**
internal gateways require an internal domain

🐰🐰🐰

### CODE REVIEW

**Comment by alii** [comment]
Would it be possible to make `internal_domain` match the camelCase spec of the other properties, then i can merge this :)

**Comment by tomheaton** [comment]
not really a fan of how I did this, open to suggestions 🐰

File: src/sdks/ignite.ts [hunk 1]
```diff
@@ -1,12 +1,6 @@
 import {create, Infer} from '@onehop/json-methods';
 import {API, assertId, Id} from '../rest/index.js';
-import {
-	Deployment,
-	DeploymentConfig,
-	Gateway,
-	GatewayType,
-	RuntimeType,
-} from '../rest/types/ignite.js';
+import {Deployment, DeploymentConfig, Gateway, GatewayType, RuntimeType} from '../rest/types/ignite.js';
```
Comment (tomheaton) [inline src/sdks/ignite.ts:3]: did not mean to do that

**Comment by alii** [comment]
this is great, thanks!

**Pull request merged**